apps/admin_app/lib/admin_app_web/templates/layout/sidebar.html.eex

```diff
@@ -11,7 +11,7 @@
           </a>
           <ul id="drilldown-1" class="nav nav-pills nav-stacked collapse pl-3">
             <li><a class="nav-link" href="#">Stock Item</a></li>
-            <li><a class="nav-link" href="#">Stock Location</a></li>
+            <li><a class="nav-link" href="/stock_locations">Stock Location</a></li>
```

Comment:
typo, need plural

Reply:
urls are always plural. Its plural only.

Reply:
I was talking about the text :slightly_smiling_face: